k9mail/src/main/java/com/fsck/k9/view/ThemeUtils.java

```diff
@@ -0,0 +1,25 @@
+package com.fsck.k9.view;
+
+
+import android.content.Context;
+import android.content.res.Resources.Theme;
+import android.support.annotation.AttrRes;
+import android.support.annotation.ColorInt;
+import android.support.annotation.ColorRes;
+import android.util.TypedValue;
+
+
+public class ThemeUtils {
+    private static final TypedValue TYPED_VALUE = new TypedValue();
+
+    @ColorInt
+    public static int getStyledColor(Context context, @AttrRes int attr) {
+        return getStyledColor(context.getTheme(), attr);
+    }
+
+    @ColorInt
+    public static synchronized int getStyledColor(Theme theme, @AttrRes int attr) {
```

Comment:
Why `synchronized`? This should only ever be called from the main thread, right?